test/TeamVesting.test.js

```diff
@@ -19,7 +19,7 @@ describe("Vesting Test", function () {
     const maxTotalSupply = 10000000000; // 10 billions
     let [owner, user1, user2, user3, user4, user5] = await ethers.getSigners();
     const SYN = await ethers.getContractFactory("SyndicateERC20");
-    const syn = await SYN.deploy(owner.address, maxTotalSupply);
+    const syn = await SYN.deploy(owner.address, maxTotalSupply, owner.address);
```

Comment:
Test it using an account different that `owner`, if not you are not really testing it.

Reply:
Oh, I see. You can remove that test completely, since it is not used right now.